package.json

```diff
@@ -17,15 +17,16 @@
   "license": "MPL",
   "devDependencies": {
     "eslint": "^8.56.0",
+    "jsdom": "^24.0.0",
```

Comment:
This is needed by vite.config.test, it's not really a change just documenting a dependency that happened to resolve before. This way we don't have to get lucky.